samples/ChatSample/Hubs/Chat.cs

```diff
@@ -5,31 +5,38 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.SignalR;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
 
 namespace ChatSample.Hubs
 {
-    // TODO: Make this work
     [Authorize]
     public class Chat : Hub
     {
+        private static readonly IDictionary<string, string> usersOnline = new ConcurrentDictionary<string, string>();
```

Comment:
I think you would store a list of connected users in Redis and you would update the list each time a user joined or left chat. Each server would subscribe to an event that would be fired when the list is modified and broadcast the change to connected users.